reference/orders.v2.oas2.yml

```diff
@@ -4532,6 +4532,7 @@ components:
             properties:
               id:
                 type: integer
+                description: 'Numeric ID of an option applied to the product from a list of options available to the product. This field has the same value as `product_option_id` when [retrieving products in an order](https://developer.bigcommerce.com/api-reference/store-management/orders/order-products/getallorderproducts).  '
```

Comment:
Double-check the spacing at the end of the sentence and make sure the apostrophes are needed here as line 3762 doesn't call for them.

Reply:
I got rid of the extra space at the end, but the single quotes are still there. any suggestions?

Reply:
^Fixed now